pkg/controller/intrusiondetection/intrusiondetection_controller_test.go

```diff
@@ -74,6 +74,7 @@ var _ = Describe("IntrusionDetection controller tests", func() {
 		mockStatus.On("IsAvailable").Return(true)
 		mockStatus.On("OnCRFound").Return()
 		mockStatus.On("ClearDegraded")
+		mockStatus.On("SetDegraded", mock.Anything, mock.Anything).Return()
```

Comment:
Doesn't this mean that tests the previously weren't setting a degraded status are now expected to, i.e. all the tests in the context "image reconciliation"? That seems like we shouldn't be changing that expectation...

Reply:
You're right, I'll throw them in a before each only for the tests that need `SetDegraded`